contracts/contracts/Sapphire.sol

```diff
@@ -23,6 +23,10 @@ library Sapphire {
         0x0100000000000000000000000000000000000007;
     address internal constant CURVE25519_PUBLIC_KEY =
         0x0100000000000000000000000000000000000008;
+    address internal constant GAS_USED =
+        0x0100000000000000000000000000000000000009;
+    address internal constant GAS_PAD =
```

Comment:
This should probably be consistent with https://github.com/oasisprotocol/oasis-sdk/blob/main/runtime-sdk/modules/evm/src/precompile/gas.rs#L23

```suggestion
    address internal constant PAD_GAS =
```